### DISCUSSION TITLE
Is there a possibility to enable the Deepin-Screen-Recorder icon in Arch Distributions?

### DISCUSSION THREAD

**Discussion by serial02** [linuxdeepin/developer-center#3560] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hi

 Is there a possibility to enable the Deepin-Screen-Recorder icon in Arch Distributions?
 The application is installed normally in Arch versions of other flavors, but it does not create an icon to be opened or added to the toolbar.

 The application only works using the terminal with the command usr/bin/deepin-screen-recorder

 See the images below:

![Captura de tela de 2023-01-29 17-53-04](https://user-images.githubusercontent.com/32621790/215356076-a2089e76-1356-484a-b173-2d1a0bb085f4.png)

![Captura de tela de 2023-01-29 17-52-57](https://user-images.githubusercontent.com/32621790/215356078-1e4f883c-c811-4442-a67b-46197677a41d.png)

![Captura de tela de 2023-01-29 17-52-16](https://user-images.githubusercontent.com/32621790/215356080-d314a337-9ddc-4555-870d-a23f378f2769.png)

![Captura de tela de 2023-01-29 17-51-37](https://user-images.githubusercontent.com/32621790/215356261-901b3cc6-48d7-40ab-be42-82baabe7062c.png)

**Comment by felixonmars** [comment]
Its desktop file has `OnlyShowIn=Deepin` set so it's hidden in other desktop environments. Since this is a standalone application I think we can lift the limit.

PR submitted: https://github.com/linuxdeepin/deepin-screen-recorder/pull/302

**Reply by serial02** [reply]
> Seu arquivo de área de trabalho tem `OnlyShowIn=Deepin`definido para que fique oculto em outros ambientes de área de trabalho. Como este é um aplicativo independente, acho que podemos aumentar o limite.
> 
> PR enviado: [ linuxdeepin/deepin-screen-recorder#302 ](https://github.com/linuxdeepin/deepin-screen-recorder/pull/302)

Thank You @felixonmars
